go.mod

```diff
@@ -51,6 +49,7 @@ replace (
 	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.17.3
 	k8s.io/sample-cli-plugin => k8s.io/sample-cli-plugin v0.17.3
 	k8s.io/sample-controller => k8s.io/sample-controller v0.17.3
+	vbom.ml/util => github.com/fvbommel/util v0.0.2 // Mitigate https://github.com/fvbommel/util/issues/6
```

Comment:
What is this dependency for?

Reply:
It is being pulled via k8s dependency chain.  Unit tests are failing atm in this repo without the fix:

```
Activated service account credentials for: [pr-kubekins@kubernetes-jenkins-pull.iam.gserviceaccount.com]
+ make test
go test -v -race ./cmd/... ./pkg/...
go: k8s.io/kubernetes@v1.17.3 requires
	k8s.io/kubectl@v0.0.0 requires
	vbom.ml/util@v0.0.0-20160121211510-db5cfe13f5cc: unrecognized import path "vbom.ml/util": https fetch: Get "https://vbom.ml/util?go-get=1": dial tcp: lookup vbom.ml on 10.63.240.10:53: no such host
make: *** [Makefile:67: test] Error 1
+ EXIT_VALUE=2
+ set +o xtrace
```

Reply:
Ref: https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/551#issuecomment-682387518